iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -123,6 +122,21 @@ enum class SubscribeState : uint32_t
     WAIT_FOR_OFFER
 };
 
+// Default properties of ChunkDistributorData
+struct DefaultChunkDistributorConfig
```

Comment:
see also below, your config for ChunkDistributor includes MAX_CHUNKS_PER_SENDER which is used in ChunkSender then. Same for MAX_CHUNKS_PER_RECEIVER  in the config for ChunkQueue. Separation of concerns here

Reply:
Yeah, got it. Put it now as separate param in chunk sender/receiver data. Have a look.